compose.yml

```diff
@@ -9,10 +9,12 @@ suite:
               bundle exec rspec"
   environment:
     - DEBUG_LEVEL=0
-    - ST2_GITURL=${ST2_GITURL}
-    - ST2_GITREV=${ST2_GITREV}
-    - ST2PKG_VERSION=${ST2PKG_VERSION}
-    - ST2PKG_RELEASE=${ST2PKG_RELEASE}
+    ## For the following variables default values are used
+    #  unless explicitly defined:
+    # - ST2_GITURL=
```

Comment:
I know. I've commented them because they generate too many unnecessary warnings. At the moment you don't need it, cuz there's no full implementation yet. So for now we live them commented.